pkg/config/config.go

```diff
@@ -0,0 +1,147 @@
+package config
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/kirsle/configdir"
+	"github.com/spf13/viper"
+)
+
+const (
+	API_VERSION     = "v1alpha1"
+	CURRENT_CONTEXT = "currentContext"
+)
+
+func Initialise() {
+	viper.SetConfigName("settings")
+	viper.SetConfigType("yaml")
+
+	viper.AddConfigPath(configdir.LocalConfig("grizzly"))
+	viper.AddConfigPath(".")
+
+	viper.BindEnv("overrides.grafana.url", "GRAFANA_URL")
+	viper.BindEnv("overrides.grafana.user", "GRAFANA_USER")
+	viper.BindEnv("overrides.grafana.token", "GRAFANA_TOKEN")
+
+	viper.BindEnv("overrides.synthetic-monitoring.token", "GRAFANA_SM_TOKEN")
+	viper.BindEnv("overrides.synthetic-monitoring.stack-id", "GRAFANA_SM_STACK_ID")
+	viper.BindEnv("overrides.synthetic-monitoring.logs-id", "GRAFANA_SM_METRICS_ID")
+	viper.BindEnv("overrides.synthetic-monitoring.metrics-id", "GRAFANA_SM_LOGS_ID")
+
+	viper.BindEnv("overrides.mimir.address", "CORTEX_ADDRESS")
+	viper.BindEnv("overrides.mimir.tenant-id", "CORTEX_TENANT_ID")
+	viper.BindEnv("overrides.mimir.api-key", "CORTEX_API_KEY")
```

Comment:
For a next PR, we could support `MIMIR_` envvars and fallback to `CORTEX_` ones

Reply:
Well, switch to calling out to mimirtool, which didn't exist when this integration was made. Then we could allow both CORTEX_ and MIMIR_ evvars, for back compat.